index.html

```diff
@@ -2,28 +2,248 @@
 <html>
 <head>
   <meta charset="utf-8">
-  <title>@brightcove/player-loader Demo: Basic</title>
+  <title>Brightcove Player Loader Demo</title>
+  <script src="/dist/brightcove-player-loader.js"></script>
+  <script src="https://unpkg.com/qs@6.5.2/dist/qs.js"></script>
+  <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
```

Comment:
Do we want these to be dev dependencies rather than relying on a cdn?

Reply:
I don't think so. I'd rather the demo page was as isolated as possible.

Reply:
ok